Sources/Profile-View.php

```diff
@@ -213,11 +213,19 @@ function summary($memID)
  */
 function fetch_alerts($memID, $all = false, $counter = 0, $pagination = array(), $withSender = true)
 {
-	global $smcFunc, $txt, $scripturl, $memberContext;
+	global $smcFunc, $txt, $scripturl, $memberContext, $user_info, $user_profile;
 
 	$query_see_board = build_query_board($memID);
 	$query_see_board = $query_see_board['query_see_board'];
 
+	// are we someone else?
+	if (empty($user_info) || $user_info['id'] !== $memID)
+	{
+		$user_old = !empty($user_info) ? $user_info : null;
+		loadMemberData($memID, false, 'minimal');
```

Comment:
Oh, I just noticed that you are calling `loadMemberData($memID, false, 'minimal')` here. It needs to be `loadMemberData($memID, false, 'profile')` in order to get the member's `time_format` value. That's probably why the errors were still happening.